core/src/utils/test/playwright/page/utils/set-content.ts

```diff
@@ -44,6 +44,7 @@ export const setContent = async (page: Page, html: string, testInfo: TestInfo, o
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
         <link href="${baseUrl}/css/ionic.bundle.css" rel="stylesheet" />
+        ${theme === 'ionic' ? `<link href="${baseUrl}/css/ionic/bundle.ionic.css" rel="stylesheet" />` : ''}
```

Comment:
This matches the [script](https://github.com/ionic-team/ionic-framework/blob/21496fb5ee45b99c4878aec5cb1c4f6320f6aa0f/core/scripts/testing/scripts.js#L40-L50) that is being used in the test pages.